tests/unit/pandas_utils/test_lusid_pandas.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import lusid
+import lusid.models as models
+import unittest
+from lusidtools.pandas_utils.lusid_pandas import lusid_respose_to_pd
+import pandas as pd
+
+test_instrument_response = lusid.models.instrument.Instrument(
```

Comment:
move these into the test class definition

Reply:
Thats done.